x-pack/plugins/ml/public/application/trained_models/models_management/expanded_row.tsx

```diff
@@ -458,65 +338,7 @@ export const ExpandedRow: FC<ExpandedRowProps> = ({ item }) => {
-                    ([pipelineName, { processors, description: pipelineDescription }]) => {
-                      return (
-                        <EuiFlexItem key={pipelineName}>
-                          <EuiPanel>
-                            <EuiFlexGroup alignItems="center" justifyContent="spaceBetween">
-                              <EuiFlexItem grow={false}>
-                                <EuiTitle size={'xs'}>
-                                  <h5>{pipelineName}</h5>
-                                </EuiTitle>
-                              </EuiFlexItem>
-                              <EuiFlexItem grow={false}>
-                                <EuiButtonEmpty
-                                  onClick={() => {
-                                    const locator = share.url.locators.get(
-                                      'INGEST_PIPELINES_APP_LOCATOR'
-                                    );
-                                    if (!locator) return;
-                                    locator.navigate({
-                                      page: 'pipeline_edit',
-                                      pipelineId: pipelineName,
-                                      absolute: true,
-                                    });
-                                  }}
-                                >
-                                  <FormattedMessage
-                                    id="xpack.ml.trainedModels.modelsList.expandedRow.editPipelineLabel"
-                                    defaultMessage="Edit"
-                                  />
-                                </EuiButtonEmpty>
-                              </EuiFlexItem>
-                            </EuiFlexGroup>
-
-                            {pipelineDescription && <EuiText>{pipelineDescription}</EuiText>}
-                            <EuiSpacer size={'m'} />
-                            <EuiTitle size={'xxs'}>
-                              <h6>
-                                <FormattedMessage
-                                  id="xpack.ml.trainedModels.modelsList.expandedRow.processorsTitle"
-                                  defaultMessage="Processors"
-                                />
-                              </h6>
-                            </EuiTitle>
-                            <EuiCodeBlock
-                              language="json"
-                              fontSize="m"
-                              paddingSize="m"
-                              overflowHeight={300}
-                              isCopyable
-                            >
-                              {JSON.stringify(processors, null, 2)}
-                            </EuiCodeBlock>
-                          </EuiPanel>
-                        </EuiFlexItem>
-                      );
-                    }
-                  )}
-                </EuiFlexGrid>
+                <ModelPipelines pipelines={pipelines!} ingestStats={stats.ingest} />
```

Comment:
Can the pipelines sections be auto-expanded by default if there are (for example) 3 or less pipelines?

![image](https://user-images.githubusercontent.com/7405507/141967849-16dced94-3778-4df0-bbbb-bbb47ee51455.png)

Reply:
Done in c3ed3e4431a3f8484edcce23546b3f86a79c8ba8